src/Installer/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -7,15 +7,23 @@
     <RootNamespace>Microsoft.DotNet.Cli.Build</RootNamespace>
     <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Nullable>annotations</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\SdkFeatureBand.cs" LinkBase="Common"/>
+    <Compile Include="$(RepoRoot)src\Common\WorkloadSetVersion.cs" LinkBase="Common" />
+  </ItemGroup>
+  
+
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="NuGet.Versioning" />
     <PackageReference Include="NuGet.Packaging" />
     <PackageReference Include="System.Reflection.Metadata" VersionOverride="$(SystemReflectionMetadataLoadContextVersion)" />
+    <PackageReference Include="Microsoft.Deployment.DotNet.Releases" />
```

Comment:
Will this trigger an additional module load for VS perf tests?

Reply:
This is in the `core-sdk-tasks` project, which is the tasks used by the .NET SDK build itself.  So it won't affect the tasks that are part of the .NET SDK and get used by most developers when building projects with the .NET SDK.  Those tasks (and the resolvers) which are loaded into VS already reference this package.